.github/pull_request_template.md

```diff
@@ -1,7 +1,33 @@
-## Overview
-<!-- Replace `XX` with the issue # you're working on -->
+# Description
+
+<!-- Please include a summary of the change(s) and which issue is being fixed. Please provide as much detail as possible. -->
+
+<!-- Replace `XX` with the concerning issue number. -->
 Closes #XX
 
-<!-- Give a brief overview of the changes made -->
+## Screenshots
+<!-- Delete this section if changes do not require screenshots -->
+
+<!-- Include any relevant screenshots or gifs to all frontend changes when applicable. -->
+
+### Before
+<!-- Add before screenshots when applicable. -->
+
+### After
+<!-- Add after screenshots when applicable. -->
+
+## Checklist
+
+- [ ] The code follows all style guidelines.
+- [ ] The code passes all required tests.
+- [ ] The code is documented.
+- [ ] The code includes tests.
+- [ ] I have self-reviewed my changes and have done QA.
+
+## General Comments
+
+<!-- Optional - Add anything else you would like to add to the Pull Request -->
+
+---
 
-<!-- Provide a screenshot of any frontend changess -->
+@VikeLabs/courseup
```

Comment:
Since all files have `@VikeLabs/courseup` as CODEOWNERS at the moment would this be redundant?

Reply:
Nvm, I thought it auto assigned the team to the PR, but it doesn't.